src/main/java/org/spongepowered/api/block/entity/Banner.java

```diff
@@ -33,7 +33,7 @@
 /**
  * Represents a Banner {@link BlockEntity}.
  */
-public interface Banner extends BlockEntity {
+public interface Banner extends BlockEntity, NameableBlockEntity {
```

Comment:
Discord comment:

Banner and enchantment table only need to extend NameableBlockEntity because of inheritance